mono/metadata/icall.c

```diff
@@ -651,6 +639,8 @@ ves_icall_System_Array_CreateInstanceImpl (MonoReflectionType *type, MonoArray *
 	return array;
 }
 
+
+// FIXME Combine ves_icall_System_Array_CreateInstanceImpl64 and ves_icall_System_Array_CreateInstanceImpl
```

Comment:
`CreateInstanceImpl64` doesn't seem to have a declaration (or callers) on the managed side. Just delete it.